gnucash/gnome/dialog-vendor.c

```diff
@@ -211,23 +211,10 @@ gnc_vendor_window_ok_cb (GtkWidget *widget, gpointer data)
 
     /* Check for valid company name */
     if (check_entry_nonempty (vw->company_entry,
-                              _("You must enter a company name. "
-                                "If this vendor is an individual (and not a company) "
-                                "you should enter the same value for:\nIdentification "
-                                "- Company Name, and\nPayment Address - Name.")))
+                              _("The Company Name field cannot be left blank,\n"
+                                "enter a company name or personal.")))
```

Comment:
I do not understand the meaning of `or personal`.

Reply:
Hi, sir
I mean "a company name or a person's name", Maybe I use a wrong word.

Reply:
Right words, wrong order. "enter a company or personal name."  Your proposal below is also fine, or you could just leave that part out. It doesn't really matter what the user puts in the Company Name field as long as it's meaningful to them. "Cash Sale" used to be common for retail sales to walk-in customers before retailers figured out that they could make money by extracting customer's names and addresses and selling the info.

Reply:
Yeah sir, Get it. Thanks for your explanation.